routers/common/repo.go

```diff
@@ -21,7 +22,56 @@ import (
 	"code.gitea.io/gitea/modules/util"
 )
 
-// ServeBlob download a git.Blob
+func setCommonHeaders(ctx *context.Context, name string, data interface{}) error {
+	// Google Chrome dislike commas in filenames, so let's change it to a space
+	name = strings.ReplaceAll(name, ",", " ")
```

Comment:
Ditto from the other PR:



I don't think this is needed actually, from what I remember about this weird bug, if you quote the filename(as being done in this function) chromium won't error about it.

Reply:
That's strange. Is it dependent on different chromium versions?

Reply:
It's a historical problem, I believe modern browsers (and modern frameworks) can handle it correctly, and we do not need hacky methods.

https://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http